go/sqlescape/ids.go

```diff
@@ -48,8 +49,48 @@ func EscapeIDs(identifiers []string) []string {
 // UnescapeID reverses any backticking in the input string.
 func UnescapeID(in string) string {
 	l := len(in)
-	if l >= 2 && in[0] == '`' && in[l-1] == '`' {
-		return in[1 : l-1]
+
+	// can't be escaped if only 2 chars
+	if l < 2 {
+		return in
+	}
+
+	// not escaped if the first and last chars aren't backticks
+	if in[0] != '`' || in[l-1] != '`' {
```

Comment:
This should be &&.

Reply:
Changed, and added an additional test that covers this.